### PR TITLE
Add SMB multichannel setting to file service

### PR DESCRIPTION
**Pull request by zfchen95**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Service team MUST add the "**WaitForARMFeedback**" label if the management plane API changes fall into one of the below categories. 
  - adding/removing APIs.
  - adding/removing properties.
  - adding/removing API-version. 
  - adding a new service in Azure.

<i>Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.</i>

- [x] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by Juliehzl** [comment]
@zfchen95 Could you add an example for the usage?

File: specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/file.json [hunk 1]
```diff
@@ -526,6 +526,10 @@
             "shareDeleteRetentionPolicy": {
               "$ref": "./common.json#/definitions/DeleteRetentionPolicy",
               "description": "The file service properties for share soft delete."
+            },
+            "protocolSettings": {
```
Comment (blueww) [inline specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/file.json:530]: Please add sample for it.
Why the bool value in such deep place? Will we add more properties in the Protocalsetting/SmbSetting/Multichannel in the near feature. we need this information in decide PSH design.

**Comment by ravbhatnagar** [comment]
New properties cant be added to existing API-version. If you need an exception, please reach out to folks from Azure API review board. Adrian Hall, Johan Stenberg, Jefferey Richter can help.

**Comment by ravbhatnagar** [comment]
Removing ARM Feedback label since ARM review board will not be reviewing property level changes going forward.

File: specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/file.json (hunk 1 above)
Comment (zfchen95) [reply specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/file.json:530]: It is to match the design of the data plane. From what I heard, there will be new protocol setting and new SMB setting in the future.

**Comment by zfchen95** [comment]
> New properties cant be added to existing API-version. If you need an exception, please reach out to folks from Azure API review board. Adrian Hall, Johan Stenberg, Jefferey Richter can help.

We understand there is a concern about breaking change with the new properties. We have a lot of practice that new properties won't be cleared by an early version SDK.

**Review by qianwens — APPROVED** [review]

**Comment by chiragg4u** [comment]
Please go through the contribution checklist added as part of the review and mark them completed.

**Comment by chiragg4u** [comment]
Please resolve breaking change as per the RPC documentation

**Comment by ravbhatnagar** [comment]
adding new properties to existing api-version is not allowed as per the Azure rest api versioning guide.

**Pull request closed without merging**